### PR TITLE
Fix import path for isPluginRequired and use correct built-ins

### PR DESCRIPTION
**Pull request by perkola**
Relates to #3671

### CODE REVIEW

**Comment by andreineculau** [comment]
I guess you'd also want to mention in package.json that this change requires `@babel/preset-env` at `^7.4.0`

**Comment by perkola** [comment]
> I guess you'd also want to mention in package.json that this change requires `@babel/preset-env` at `^7.4.0`

Good catch! I was a bit too quick. I have updated the PR now.

File: packages/@vue/babel-preset-app/index.js [hunk 1]
```diff
@@ -14,8 +14,8 @@ const defaultPolyfills = [
 ]
 
 function getPolyfills (targets, includes, { ignoreBrowserslistConfig, configPath }) {
-  const { isPluginRequired } = require('@babel/preset-env')
-  const builtInsList = require('@babel/preset-env/data/built-ins.json')
+  const { isPluginRequired } = require('@babel/preset-env/lib/filter-items')
+  const builtInsList = require('@babel/preset-env/data/corejs2-built-ins.json')
```
Comment (nicolo-ribaudo) [inline packages/@vue/babel-preset-app/index.js]: Hi, Babel maintainer here 👋

We just released v7.4.1, which should fix the issue. The `builtInsList` change is correct, but I'd ask you to revert line 17 (it's better not to rely on internal files, when possible). You'll need to update the `preset-env` version in package.json.

**Comment by perkola** [comment]
This PR becomes obsolete with the release of `@babel/preset-env` `v.7.4.1`.

File: packages/@vue/babel-preset-app/index.js (hunk 1 above)
Comment (perkola) [reply packages/@vue/babel-preset-app/index.js]: Hi! Thank you for the quick reply. I noticed that you also aliased `cosejs2` so both my changes are unnecessary right now. Perhaps we still need to change the path of `built-ins.json` if the alias is removed in the future.

**Comment by nicolo-ribaudo** [comment]
Please update `package.json` to `^7.4.1` to avoid accidentally downloading `7.4.0`

**Pull request closed without merging**